bots/employee-finder/src/SSW.SophieBot/Integration/SophieBotAdapter.cs

```diff
@@ -36,7 +36,10 @@ public class SophieBotAdapter : CloudAdapter
 		protected virtual async Task OnTurnErrorAsync(ITurnContext turnContext, Exception exception)
 		{
 			var properties = new Dictionary<string, string>
-				{{"Bot exception caught in", $"{nameof(SophieBotAdapter)} - {nameof(OnTurnError)}"}};
+			{
+				{ "Bot exception caught in", $"{nameof(SophieBotAdapter)} - {nameof(OnTurnError)}" },
+				{ "Question", turnContext.Activity.Text },
```

Comment:
Tested locally. We can get the question from `turnContext.Activity.Text`

<img width="938" alt="image" src="https://github.com/SSWConsulting/SSW.SophieBot/assets/37203901/7854ac1f-eed3-40db-957b-7f573f4d4f97">